docs/typelibrary/utils/sequence-control/combi/sequence_E_04.md

```
@@ -0,0 +1,99 @@
+# sequence_ET_04
```

markdownlint-cli2 (0.17.2)

Required heading structure
Expected: # Head; Actual: # sequence_ET_04

(MD043, required-headings)